crates/aquascope/src/analysis/permissions/context.rs

```diff
@@ -299,6 +302,7 @@ impl<'tcx> PermissionsCtxt<'tcx> {
       path_uninitialized: false,
       loan_read_refined: None,
```

Comment:
Do we still need the two `loan_(read|write)_refined` fields now that we have a `loan_refined` field?